Firmware/MVP/BringupTester.cpp

```diff
@@ -50,18 +50,18 @@ void bringupTester_run(void)
 				Serial.print(F("\nLTC6804 - isoSPI error count is "));
 				Serial.print(String(LTC68042result_errorCount_get()));
 				Serial.print(F(": "));
-			
+
 				if( LTC68042result_errorCount_get() == 0 ) { Serial.print(F("pass")); }
-				else { Serial.print(F("FAIL!!!!!!!!!!!!!")); didTestFail = true; } //at least one isoSPI error occurred
+				else { Serial.print(F("FAIL!! !! !! !! !")); didTestFail = true; } //at least one isoSPI error occurred
```

Comment:
Ha, that's a silly Arduino bootloader bug!
Did you actually run into this with LiBCM?
Regardless, yes, please keep your new formatting ("!! !! !!"), so that we don't run into this in the future.